Makefile

```diff
@@ -1,20 +1,29 @@
 PREFIX?=/usr/local
 
+TEMPORARY_FOLDER=./tmp_portable_licenseplist
+
 build:
 	swift build -c release -Xswiftc -static-stdlib
- 
-test: 
+
+test:
 	swift test
 
 lint:
 	swiftlint
- 
+
 clean:
 	swift package clean
 
 xcode:
 	swift package generate-xcodeproj
-	
+
 install: build
 	mkdir -p "$(PREFIX)/bin"
 	cp -f ".build/release/LicensePlist" "$(PREFIX)/bin/license-plist"
+
+portable_zip: build
+	mkdir "$(TEMPORARY_FOLDER)"
+	cp -f ".build/release/LicensePlist" "$(TEMPORARY_FOLDER)"
```

Comment:
Should be `cp -f ".build/release/LicensePlist" "$(TEMPORARY_FOLDER)/license-plist"`?